packages/common/src/ui/components/buttons/IconButton.tsx

```diff
@@ -1,36 +1,33 @@
 import React from 'react';
-import { Button, Tooltip } from '@mui/material';
-import { styled } from '@mui/material/styles';
+import { IconButton as MuiIconButton, Tooltip } from '@mui/material';
```

Comment:
I made the `IconButton` into `ButtonWithIcon` and `UnstyledIconButton` into `IconButton` as it made more sense to me at the time..